deploy/docker/uwsgi.ini

```diff
@@ -6,7 +6,10 @@ http-socket = 0.0.0.0:3031
 #chmod-socket = 664
 master = true
 cheaper = 2
-processes = 4
+# %k is detected cpu cores
+processes = %(%k * 2 + 1)
```

Comment:
Just wondering where `2n+1` came from?

Also we could try using this https://uwsgi-docs.readthedocs.io/en/latest/Cheaper.html so we don't waste resources while not under heavy load

Reply:
Sorry, meant that we could use the memory limit and step specifically (since we are already using cheaper)